pkg/service/test/test.go

```diff
@@ -274,7 +275,13 @@ func (t *tester) Test(path string, testReportPath string, generateTestReport boo
 		testSuiteNames = append(testSuiteNames, testSuiteName)
 	}
 
-	sort.Strings(testSuiteNames)
+	sort.SliceStable(testSuiteNames, func(i, j int) bool {
+
+        num1, _ := strconv.Atoi(strings.Split(testSuiteNames[i], "-")[2])
```

Comment:
To make it not throw error in case someone updates the` testSuiteNames` better to put a length check as well as address the errors from `strconv.Atoi` (returning false in the case would be a good idea)

Reply:
what should be the max length?

Reply:
so when you split the testsuitename it goes into array of 3 (as of now expected format is `test-set-X` thus its fair to check if length is 3 atleast as to order the last element X of the name.